stdlib/string.gr

```diff
@@ -454,9 +476,9 @@ provide let split = (separator: String, string: String) => {
       if (Memory.compare(ptr, pptr, psize) == 0n) {
         let strSize = ptr - last
         let str = allocateString(strSize)
-        Memory.copy(str + 8n, last, strSize)
-        WasmI32.store(arr + arrIdx, str, 8n)
-        arrIdx += 4n
+        Memory.copy(str + _STR_DATA_OFFSET, last, strSize)
+        WasmI32.store(arr + _ARRAY_DATA_OFFSET, str, arrIdx)
```

Comment:
Why did you flip these

Reply:
I flipped them as I figured we should be offsetting into the array for the item rather than moving the pointer, it doesn't really make a difference though.